python/tank/descriptor/bundle_cache_usage/writer_base.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+
+class BundleCacheUsageWriterBase(object):
```

Comment:
this feels overengineered to me. Why do we need a base class when we only have a single class deriving from it an no future plans to add any additional implementations?

Reply:
No prob, dropping it. I have an uncommited plain text file writer. Something we discussed in the very beginnig hence the 'base class'